docs/source/getting_started/command_line_tool.rst

```diff
@@ -119,16 +119,27 @@ You can upload embeddings directly to the Lightly Platform using the CLI.
 Download data using the CLI
 -----------------------------------------------
 You can download a dataset with a given tag from the Lightly Platform using the 
-following CLI command. The CLI provides you with two options. Either you 
-download just a list or copy the files from the original dataset into a new 
-folder. The second option is very handy for quick prototyping.
+following CLI command. The CLI provides you with three options:
+
+* Download the list of filenames for a given tag in the dataset.
+  
+* Download the images for a given tag in the dataset.
```

Comment:
Here I would add a warning that downloading all images might take long. I prefer saying "Option 2 has the disadvantage that ..." to saying "Option 3 has the advantage of not having the disadvantage of option 2".

Reply:
I like it better this way, has a more positive tone.